src/cli/commands/init.js

```diff
@@ -114,14 +121,18 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       def = val;
     }
 
+    if (manifestKey === 'private' && privateFlag) {
+      def = true;
+    }
+
     if (def) {
-      question += ` (${def})`;
+      question += ` (${def.toString()})`;
```

Comment:
If removed it will generate 2 errors:
```
$ eslint . && flow check
Error: src/cli/commands/init.js:130
130:       question += ` (${def})`;
                            ^^^ boolean. This type cannot be coerced to
130:       question += ` (${def})`;
                       ^^^^^^^^^^^ string
```